Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -1452,24 +1452,26 @@ protected function reflectClassMethodParameter($className, MethodReflection $met
      */
     protected function expandType(ClassReflection $class, $type)
     {
+        $typeWithoutNull = TypeHandling::stripNullableType($type);
+        $isNullable = $typeWithoutNull !== $type;
```

Comment:
..could then be changed to `TypeHandling::isNullableType($targetType)` in the places below